packages/jss-plugin-syntax-expand/.size-snapshot.json

```diff
@@ -8,5 +8,15 @@
     "bundled": 11240,
     "minified": 4432,
     "gzipped": 1544
+  },
+  "dist/jss-plugin-syntax-expand.js": {
```

Comment:
wrong name

Reply:
ah its a change from a different package here

Reply:
Yeah, forgot to build and it created them only now with the correct names.